build.properties

```diff
@@ -14,6 +14,7 @@ username=test
 password=test
 privilegedUser=postgres
 privilegedPassword=
+gssEncMode=disable
```

Comment:
Is it better to stick with default mode (which seems to be `prefer`)?

Reply:
ya we can remove this.

Reply:
I would suggest to keep it as `gssEncMode=prefer`, so it is aligned with the default behavior. It would be easier to discover and test if the property is listed here.